src/index.ts

```diff
@@ -153,6 +155,17 @@ export async function main(
   let requestedPaths: string[] = []
   
   // Look for files
+
+  if (args.gcov) {
+    UploadLogger.verbose('Running gcov...')
+    try {
+      const gcovLogs = await generateGcovCoverageFiles(projectRoot)
+      UploadLogger.verbose(`${gcovLogs}`)
+    }
+    catch (error) {
+      throw new Error(`${error}`)
+    }
```

Comment:
```suggestion
    const gcovLogs = await generateGcovCoverageFiles(projectRoot)
    UploadLogger.verbose(`${gcovLogs}`)
```

Allow the original error to fall through to the appropriate error handler.

Reply:
Hmm earlier I was getting an UnhandledPromiseRejection which is why I added this.  But with the test changes that seems to have disappeared